google/cloud/spanner/internal/connection_impl_test.cc

```diff
@@ -2006,9 +2006,9 @@ TEST(ConnectionImplTest, ExecutePartitionedDmlRetryableInternalErrors) {
       .WillOnce(Return(ByMove(MakeReader(
           {}, {grpc::StatusCode::INTERNAL,
                "Received unexpected EOS on DATA frame from server"}))))
-      .WillOnce(
-          Return(ByMove(MakeReader({}, {grpc::StatusCode::INTERNAL,
-                                        "HTTP/2 error code: INTERNAL_ERROR"}))))
+      .WillOnce(Return(ByMove(MakeReader(
+          {}, {grpc::StatusCode::INTERNAL,
+               "HTTP/2 error code: INTERNAL_ERROR\nReceived Rst Stream"}))))
```

Comment:
Opt: Perhaps a newline in an error message isn't the best choice?

Reply:
@devbww and I talked offline. While it is weird to have a newline in the error message, we are not the ones creating this error message. We are just handling it. So it's fine to include a newline in our tests.